lib/discovery.ex

```diff
@@ -39,10 +39,16 @@ defmodule Onvif.Discovery do
       {:udp, _port, device_ip, device_port, udp_response} ->
         probe_response = parse_udp_xml_response(udp_response)
         string_device_ip = device_ip |> :inet.ntoa() |> List.to_string()
+        device_addr = "http://#{string_device_ip}:#{device_port}"
```

Comment:
this is just the port from the WS probe response, it won't actually have anything to do with onvif and won't be a valid address. The UDP ports won't response to HTTP calls